drivers/ieee802154/Kconfig.dw1000

```diff
@@ -49,4 +49,11 @@ config ENABLE_MANUAL_TX_POWER_CONTROL
 	  increase the transmit power and still remain within regulatory power limits
 	  which are typically specified as average power per millisecond.
 
+config PRF_64MHZ
```

Comment:
Same as the Kconfig option introduced in the previous patch, but the effect here is more problematic.

Reply:
See previous comment re Kconfig vs DT, BUT
- please provide this as an option to choose from for improved readability and more importantly the standard provides more than two options and future qorvo chipsets may very well support them.
- All options in this file must be properly prefixed. Please use the prefix introduced for the other options in this file.

Reply:
Options are now prefixed.
PRF is now a choice.